src/gtc/oir.py

```diff
@@ -120,13 +120,18 @@ class ScalarDecl(Decl):
     pass
 
 
+class LocalScalar(ScalarDecl):
+    pass
+
+
 class Temporary(FieldDecl):
```

Comment:
```suggestion
class Temporary(Decl):
```
because otherwise we can pass a temporary as param to the stencil

Reply:
Done.